lib/queue.js

```diff
@@ -337,18 +339,29 @@ Queue.prototype.close = function( doNotWaitJobs ){
 
   @method process
 */
-Queue.prototype.process = function(concurrency, handler){
+Queue.prototype.process = function(options, handler){
   var _this = this;
```

Comment:
The major problem with breaking the API is that it will require a new major release of bull (3.0.0), so it would be better if we could keep backwards compatibility. Maybe it could be possible to put it in the redisOpts in the constructor as a transition solution, and later move the redis opts inside a redis object (inside the more generic opts).